### DISCUSSION TITLE
Pagination within subrows.

### DISCUSSION THREAD

**Discussion by ShivankZeda** [TanStack/table#3940] [Q&A]
Hey team, great product !! Small implementation where I am stuck. So i want to use the expanded subrows implementation, but as there can be a large amount of subrows, it has to paginated which is driven by the scroll bar within the subrow group. As subrows are also considered as a child row of the table, i am not able to add a scroll for the subrow group. Can you please help or guide me with the implementation.

**Comment by tannerlinsley** [comment]
Turn on manualExpanding and build your own expanded row model to keep expanded rows nested. This way you can wrap them in a scroll.

**Reply by ShivankZeda** [reply]
Thanks for your prompt response ! Will give it a go and update you.

**Reply by Chrisdo82** [reply]
@ShivankZeda did you find a solution to this?
